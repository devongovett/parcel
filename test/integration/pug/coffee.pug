doctype html
html(lang="en")
  head
    title= pageTitle
    script
      :coffee-script
        console.log 'This is coffee script'
