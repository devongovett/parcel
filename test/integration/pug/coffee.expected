<!DOCTYPE html>
<html lang="en">
  <head>
    <title></title>
    <script>(function() {
  console.log('This is coffee script');

}).call(this);

    </script>
  </head>
</html>
